libs/components/src/lib/dial-pad/dial-pad.scss

```
@@ -0,0 +1,40 @@
+@use '../../../../../dist/libs/tokens/scss/tokens.constants' as constants;
+@use '../../../../shared/src/lib/sass/mixins/border-radius' as
+	border-radius-variables;
+
+$inline-size: 230px;
+$gap: 16px;
+
+.base {
+	display: grid;
+	margin: 16px;
+	gap: $gap;
```

```suggestion
 grid-template-rows: 80px 1fr auto;
```
this way - there will be no layout change when the error text or helper text is added (as long as it is max 2 lines there)